mex/common/wikidata/extract.py

```diff
@@ -46,6 +46,42 @@ def search_organization_by_label(
     return _get_organization_details(wd_item_id)
 
 
+def get_count_of_found_organizations_by_label(
+    item_label: str,
+    lang: TextLanguage,
+) -> int:
+    """Get total count of searched organizations in wikidata.
+
+    Args:
+        item_label: Item title or label to be searched
+        lang: language of the label. Example: en, de
+
+    Returns:
+        list of WikidataOrganization
```

Comment:
not quite

Reply:
damn it, always the docs. updated in: 7ccd07a34289a2e3cff34c8bd42821ab147d633f